### PR TITLE
Move imputation method evaluation to own directory and add README

### LINKED ISSUE #12: [imputation_method_eval] Move the analyses in top directory to their own folder and summarize results in README
These are the analyses that specifically dealt with masking data and evaluating imputation methods' performance.

### PR DESCRIPTION
**Pull request by jaclyn-taroni**
Closes #12. Also changes the paths to the tximport-process RNA-seq sample files in the relevant `quality_check` notebooks. Note that the plot won't show up in the README until this gets merged.

### CODE REVIEW

File: select_imputation_method/README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Selecting an imputation method for refine.bio species compendia
+
+### Background
+
+[refine.bio](https://github.com/AlexsLemonade/refinebio) contains gene expression data measured with multiple technologies and many platforms.
```
Comment (cansav09) [inline select_imputation_method/README.md]: Is there an extra space after the parenthesis or am I just imagining it?

File: select_imputation_method/README.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+# Selecting an imputation method for refine.bio species compendia
+
+### Background
+
+[refine.bio](https://github.com/AlexsLemonade/refinebio) contains gene expression data measured with multiple technologies and many platforms. 
+For instance, there are over 25 refine.bio-supported microarray platforms for human assays alone. 
+Microarrays, which currently comprise the bulk of publicly available gene expression data, are limited by the sequences contained on the array itself; 
+older arrays measure fewer genes compared to more recent platforms.
```
Comment (cansav09) [inline select_imputation_method/README.md]: If this is the same sentence as the semicolon indicates, you can leave it on the same line right?

File: select_imputation_method/README.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+### Background
+
+[refine.bio](https://github.com/AlexsLemonade/refinebio) contains gene expression data measured with multiple technologies and many platforms. 
+For instance, there are over 25 refine.bio-supported microarray platforms for human assays alone. 
+Microarrays, which currently comprise the bulk of publicly available gene expression data, are limited by the sequences contained on the array itself; 
+older arrays measure fewer genes compared to more recent platforms. 
+
+The goal of refine.bio is to provide uniformly processed and normalized gene expression data. 
+This includes the capability to combine samples that were measured on different technologies. 
+In many cases, removing genes that not measured for every sample will result in a greatly reduced feature space.
```
Comment (cansav09) [inline select_imputation_method/README.md]:
```suggestion
In many cases, removing genes that are not measured for every sample will result in a greatly reduced feature space. 
```

File: select_imputation_method/README.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+The goal of refine.bio is to provide uniformly processed and normalized gene expression data. 
+This includes the capability to combine samples that were measured on different technologies. 
+In many cases, removing genes that not measured for every sample will result in a greatly reduced feature space. 
+To avoid this problem, it is desirable to impute missing values. 
+Here, we're evaluating imputation approaches for constructing species compendia. Species compendia will be comprised of all samples from a species in our system.
+
+### Experiment
+
+We constructed gene expression matrix from ~800 _Danio rerio_ samples split approximately evenly between microarray and RNA-seq data and only retained genes measured in all samples.
```
Comment (cansav09) [inline select_imputation_method/README.md]:
```suggestion
We constructed a gene expression matrix from ~800 _Danio rerio_ samples split approximately evenly between microarray and RNA-seq data and only retained genes measured in all samples.
```

File: select_imputation_method/README.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+The rationale for the log2 transformation was to reduce the effect of outliers. 
+In practice, we aim to have the distributions between samples as similar as possible so we probably want quantile normalization to be the _last_ step of any species compendium processing.
+
+* `KNNImputer` did not perform particularly well. 
+Many of the imputed values were 0 and it was unlikely to be appropriate for the number of samples that will have (see [this Bitbucket issue](https://bitbucket.org/libsleipnir/sleipnir/issues/28/segmentation-fault-with-100-000-samples)).
+
+* `KNN` performed well and has been regularly used for imputing gene expression values ([Troyanskaya et al., 2001](https://doi.org/10.1093/bioinformatics/17.6.520)), which is why we elected to explore it further.
+Unfortunately, it would likely take too long to run (see [`impute_requirements`](https://github.com/AlexsLemonade/compendium-processing/tree/master/impute_requirements))!
+
+Ultimately, we elected to use `IterativeSVD` for initial species compendia processing.
```
Comment (cansav09) [inline select_imputation_method/README.md]: Need extra space. Also can you put a sentence about why `IterativeSVD`?

File: select_imputation_method/README.md (hunk 2 above)
Comment (jaclyn-taroni) [reply select_imputation_method/README.md]: You can but I think it's easier to review/comment on the text this way with no adverse effect on the rendered markdown.

File: select_imputation_method/README.md (hunk 1 above)
Comment (jaclyn-taroni) [reply select_imputation_method/README.md]: No extra space

**Pull request merged**